auth-web/src/locales/en.json

```diff
@@ -206,10 +206,10 @@
   "removeAmalgamationSuccessText": "You have successfully deleted an amalgamation application",
   "removeAmalgamationConfirmTitle": "Delete Amalgamation Application?",
   "removeAmalgamationConfirmText": "Deleting this amalgamation application will remove the application from your Business Registry list. If this amalgamation application was associated with a Name Request, the Name Request can still be used to start an amalgamation application.",
-  "removeContinuationInSuccessTitle": "Continuation In Deleted",
-  "removeContinuationInSuccessText": "You have successfully deleted a continuation in",
-  "removeContinuationInConfirmTitle": "Delete Continuation In?",
-  "removeContinuationInConfirmText": "Deleting this continuation in will remove the application from your Business Registry list. If this continuation in was associated with a Name Request, the Name Request can still be used to start a continuation in.",
+  "removeContinuationInSuccessTitle": "Continuation In Application Deleted",
+  "removeContinuationInSuccessText": "You have successfully deleted a continuation in application",
+  "removeContinuationInConfirmTitle": "Delete Continuation In Application?",
+  "removeContinuationInConfirmText": "Deleting this continuation in application will remove the application from your Business Registry list. If this continuation in was associated with a Name Request, the Name Request can still be used to start a continuation in.",
```

Comment:
"continuation in" is used twice more in this paragraph. Do those need to be updated?

Generally, did you chat with Yui about all of these and which ones need to be updated?

Reply:
I will double check with him